projects/components/src/table/table.component.ts

```diff
@@ -561,14 +579,52 @@ export class TableComponent
     };
   }
 
-  @HostListener('mousemove', ['$event'])
   public onResizeMouseMove(event: MouseEvent): void {
-    this.debouncedResizeColumn(event);
+    if (this.resizable && this.resizeStartX > 0 && !isNil(this.columnResizeHandler)) {
+      this.columnResizeHandler.style.right = `${this.resizeStartX - event.clientX}px`;
+    }
+  }
+
+  public onResizeMouseUp(event: MouseEvent): void {
+    this.checkAndResizeColumn(event);
+    this.changeDetector.detectChanges();
+  }
+
+  private addEventListeners(): void {
+    this.document.addEventListener('mousemove', event => this.onResizeMouseMove(event));
+    this.document.addEventListener('mouseup', event => this.onResizeMouseUp(event));
+  }
+
+  private removeEventListeners(): void {
+    this.document.removeEventListener('mousemove', event => this.onResizeMouseMove(event));
+    this.document.removeEventListener('mouseup', event => this.onResizeMouseMove(event));
   }
 
-  @HostListener('mouseup')
-  public onResizeMouseUp(): void {
-    this.resizeColumns = undefined;
+  private checkAndResizeColumn(event: MouseEvent): void {
```

Comment:
can we do auto fit content for all columns?

Reply:
discussed offline